include/grpcpp/impl/codegen/core_codegen_interface.h

```diff
@@ -19,6 +19,8 @@
 #ifndef GRPCPP_IMPL_CODEGEN_CORE_CODEGEN_INTERFACE_H
 #define GRPCPP_IMPL_CODEGEN_CORE_CODEGEN_INTERFACE_H
 
+#include <cassert>
```

Comment:
Why is this inclusion needed now when it wasn't used before?

Reply:
NDEBUG needs it right?

Reply:
or I guess we could depend on the users of this to include it instead. That's what Core does right now

Reply:
Ah ofcourse! you are right. That was my moment of stupidity :D
Removed it